helm/flink-kubernetes-operator/values.yaml

```diff
@@ -30,6 +30,9 @@ image:
 rbac:
   create: true
 
+operatorPodDeployment:
```

Comment:
I find the name confusing. Is it for the pod or the deployment?

Reply:
it is for Pod annotation. I had named it this way, as we have only a helm template for deployment, but we can change it. Please let me know, if there is any suggestion

Reply:
Then we could call it simply `operatorPod`